db/schema.rb

```diff
@@ -35,8 +35,8 @@
   create_table "activities", force: :cascade do |t|
     t.integer  "user_id"
     t.string   "action"
-    t.integer  "actionable_id"
     t.string   "actionable_type"
+    t.integer  "actionable_id"
```

Comment:
Could you regenerate the `schema.rb` file with only the changes to the tables we are updating in this PR? 🙏